cruise.umple/test/cruise/umple/implementation/java/ClassTemplateTest_InternalConstant.java.txt

```diff
@@ -0,0 +1,34 @@
+/*PLEASE DO NOT EDIT THIS CODE*/
+/*This code was generated using the UMPLE 1.26.0-b05b57321 modeling language!*/
+
+
+
+// line 1 "test.ump"
+public class Student
+{ 
+  //------------------------
+  // STATIC VARIABLES
+  //------------------------
+ 
+private static final String Name = "XYZ";
```

Comment:
All this seems to be testing is the use of extra code

Reply:
Same mistake as described above